twitter.html

```diff
@@ -0,0 +1,327 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Twitter</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+    <div class="container">
+<!-- LEFT MENU -->
+        <div class="left-menu">
+
+            <a class="twitter-logo" href="#"><img src="./images/SVG's/twitter-color-svgrepo-com.svg" alt="Twitter logo"
+                    width="30px"></a>
+
+            <svg fill="#1DA1F2" height="26.25px" width="26.25px" version="1.1" xmlns="http://www.w3.org/2000/svg"
+                xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 24 24" xml:space="preserve">
+                <g>
+                    <path
+                        d="M12 1.696L.622 8.807l1.06 1.696L3 9.679V19.5C3 20.881 4.119 22 5.5 22h13c1.381 0 2.5-1.119 2.5-2.5V9.679l1.318.824 1.06-1.696L12 1.696zM12 16.5c-1.933 0-3.5-1.567-3.5-3.5s1.567-3.5 3.5-3.5 3.5 1.567 3.5 3.5-1.567 3.5-3.5 3.5z">
+                    </path>
+                </g>
+            </svg>
+            <a class="nav" href="#">
+                <h2>Home</h2>
+            </a>
+
+            <img src="./images/SVG's/hash-svgrepo-com.svg" alt="" width="30px">
+            <a class="nav" href="#">
+                <h2>Explore</h2>
+            </a>
+
+            <img src="./images/SVG's/bell-svgrepo-com.svg" alt="Notification Bell" width="30px">
+            <a class="nav" href="#">
+                <h2>Notifications</h2>
+            </a>
+
+            <img src="./images/SVG's/mail-svgrepo-com.svg" alt="" width="30px">
+            <a class="nav" href="#">
+                <h2>Messages</h2>
+            </a>
+
+            <img src="./images/SVG's/bookmark-svgrepo-com.svg" alt="" width="30px">
+            <a class="nav" href="#">
+                <h2>Bookmarks</h2>
+            </a>
+
+            <img src="./images/SVG's/file-06-svgrepo-com.svg" alt="" width="30px">
+            <a class="nav" href="#">
+                <h2>Lists</h2>
+            </a>
+
+            <img src="./images/SVG's/profile-round-1342-svgrepo-com.svg" alt="" width="30px">
+            <a class="nav" href="#">
+                <h2>Profile</h2>
+            </a>
+
+            <img src="./images/SVG's/more-horizontal-circle-svgrepo-com.svg" alt="" width="30px">
+            <a class="nav" href="#">
+                <h2>More</h2>
+            </a>
```

Comment:
Maybe you put all these into individual divs ?

Reply:
Agreed. The grid worked without it though, so I left it.